src/loadScript.test.ts

```diff
@@ -0,0 +1,66 @@
+import { loadScript } from './loadScript';
+
+const goodURL = 'good-url';
+const badURL = 'bad-url';
+
+const getTestScript = (url: string) =>
+	Array.from(document.scripts).find(({ src }) => src.includes(url));
+
+const fakeLoadEvent = new MutationObserver((mutations) => {
```

Comment:
Could you add a comment to this, as it took me a while to reason with what's happening with MutationObserver